app/scripts/templates/settings/client_disconnect.mustache

```diff
@@ -0,0 +1,40 @@
+<div id="client-disconnect">
+  <section class="modal-panel">
+    {{^hasDisconnected}}
+      <h2>{{#t}}Disconnect from Sync?{{/t}}</h2>
+      <div class="error"></div>
+
+      <p class="intro">
+      {{#t}}Your browsing data will remain on %(deviceName)s, but it will no longer sync with your account.{{/t}}
+      </p>
+
+      <form novalidate>
+
+        <div class="select-row-wrapper">
+        <label class="label-helper"></label>
+        <div class="select-row">
+          <select class="disconnect-reasons">
+            <option value="reason">{{#t}}Reason for disconnecting...{{/t}}</option>
```

Comment:
Placeholder text does not disappear when going back to default option. Is this intended, I can see it either way.
![screen shot 2016-09-19 at 4 11 56 pm](https://cloud.githubusercontent.com/assets/1295288/18647387/beea3e06-7e83-11e6-9ddf-d902eb1fe178.png)